lib/services/ios-project-service.ts

```diff
@@ -1312,6 +1314,23 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		}
 		return teamId;
 	}
+
+	private validateApplicationIdentifier(projectData: IProjectData): void {
+		const projectDir = projectData.projectDir;
+		const infoPlistPath = path.join(projectDir, constants.APP_FOLDER_NAME, constants.APP_RESOURCES_FOLDER_NAME, this.getPlatformData(projectData).normalizedPlatformName, this.getPlatformData(projectData).configurationFileName);
+		const mergedPlistPath =  this.getPlatformData(projectData).configurationFilePath;
+
+		if (!this.$fs.exists(infoPlistPath) || !this.$fs.exists(mergedPlistPath)) {
+			return;
```

Comment:
maybe some trace message would be useful here, so we figure out what's wrong if the user has problems